src/Button/_variables.scss

```diff
@@ -43,3 +43,8 @@ $btn-border-radius-lg:        $border-radius-lg !default;
 $btn-border-radius-sm:        $border-radius-sm !default;
 
 $btn-transition: null;
+
+$distance: $btn-focus-width + $btn-focus-gap !default;
+$distance-include-btn-border: $distance + $btn-border-width !default;
+
+$btn-border-radius-value: calc(#{$btn-border-radius} + #{$distance}) !default;
```

Comment:
can we rename this to something more obvious? right now it's not clear what this is used for, especially since we already have `$btn-border-radius` variable. I think this one is used only during focus state, maybe name it something like `btn-focus-border-radius`?

Also, same comment to `distance` variables above, it's not clear what distance they represent, maybe rename them to something like `btn-focus-distance-to-border`? (I think they represent distance to broder during focus state, but maybe I'm wrong)

Reply:
Corrected, thanks